package.json

```diff
@@ -63,8 +63,8 @@
     "find-npm-prefix": "^1.0.2",
     "fs-extra": "^11.2.0",
     "gittar": "^0.1.1",
-    "mina-signer": "^2.1.1",
-    "o1js": "^0.15.*",
+    "mina-signer": "^2.1.2",
+    "o1js": "^0.15.4",
```

Comment:
```suggestion
     "o1js": "^0.15.*",
```

Reply:
Is there a reason you you changed this? We added the pattern  `"o1js": "^0.15.*"` so we only have to update the cli and push a new version for every minor release of o1js and not every patch release as well.

Reply:
https://github.com/o1-labs/zkapp-cli/pull/564#discussion_r1471837996